doc/deploy.md

```diff
@@ -1,49 +1,48 @@
-# How to deploy a local environment
+# Deploy an xmtpd node to XMTP testnet
```

Comment:
I updated this title - but I am still not sure I understand that this document is about. =P

It seems like it is about deploying an xmtpd node to testnet - but it also includes info about deploying a local environment. I think what I need to understand is how these two things work together.

My guess is that the main topic of this doc is how to deploy an xmtpd node to testnet - if you are an xmtpd dev and need to do this for an Ephemera-run node.

But this doc is also about how to deploy an xmtpd node if you need to use it with a local dev env -- but the node is not on the testnet?

Reply:
Thank you @fbac - love it - let me take another pass at it!